bench/tx-generator/src/Cardano/Benchmarking/Script/Core.hs

```diff
@@ -477,15 +476,22 @@ createChangeInEra value count _proxy = do
 
 createChangeGeneric :: ([Lovelace] -> ActionM (Either String (TxInMode CardanoMode))) -> Lovelace -> Int -> ActionM ()
 createChangeGeneric createCoins value count = do
+  submitTracer <- btTxSubmit_ <$> get BenchTracers
   let
     coinsList = replicate count value
     maxTxSize = 30
     chunks = chunkList maxTxSize coinsList
+    msg = mconcat [ "createChangeGeneric: outputs: ", show count
+                  , " value: ", show value
+                  , " number of txs: ", show $ (count + maxTxSize -1) `div` maxTxSize
```

Comment:
Hmm, but why do we have to re-compute the implicitly available data?

Isn't the `nTxs == length chunks`?

Reply:
Ahh hmm yes OFC it is